edge/pkg/edged/edged_pods.go

```diff
@@ -1438,3 +1460,29 @@ func (pk *podKillerWithChannel) PerformPodKillingWork() {
 		}
 	}
 }
+
+// getHostIPByInterface gets the IP address of a the custom network interface
+func (e *edged) getHostIPByInterface() (string, error) {
```

Comment:
Seems the method is not used :)

Reply:
Not anymore, already deleted locally. Will be removed with the next commit.